Mathlib/MeasureTheory/MeasurableSpace/CountablyGenerated.lean

```diff
@@ -334,11 +334,11 @@ lemma measurableSet_generateFrom_memPartition_iff (t : ℕ → Set α) (n : ℕ)
       ↔ ∃ S : Finset (Set α), ↑S ⊆ memPartition t n ∧ s = ⋃₀ S := by
   refine ⟨fun h ↦ ?_, fun ⟨S, hS_subset, hS_eq⟩ ↦ ?_⟩
   · refine MeasurableSpace.generateFrom_induction
```

Comment:
Can you use `induction` or `induction'` here?

Reply:
I didn't want to do without being asked, because it would increase the diff. But I agree it's a good idea